src/gio_future.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use futures_channel::oneshot;
+use futures_core::task::Context;
+use futures_core::{Async, Future};
+
+use glib;
+use Cancellable;
+
+/// Represents a GIO asynchronous operation as a future
```

Comment:
This comment and for new better removed and added to lgpl-docs later

Reply:
Why?

Reply:
Because travis don't allow class and function comment in gio at all.

Reply:
Can't we just disable that? We might want to add more docs in the future, and this is not really part of the gir-extracted docs.

Reply:
We have https://raw.githubusercontent.com/gtk-rs/lgpl-docs/master/gio/docs.md specially for manual comments.

Reply:
Done